src/style.css

```diff
@@ -0,0 +1,50 @@
+body {
+  margin: 0;
+}
+
+h1 {
+  display: none;
+}
```

Comment:
That's a bad idea to remove all `h1` from the page, just remove the correspondent `h1` in your `index.html`